cylc/flow/etc/job.sh

```diff
@@ -73,17 +73,16 @@ cylc__job__main() {
     export CYLC_WORKFLOW_LOG_DIR="${CYLC_WORKFLOW_RUN_DIR}/log/workflow"
     export CYLC_WORKFLOW_SHARE_DIR="${CYLC_WORKFLOW_RUN_DIR}/share"
     export CYLC_WORKFLOW_WORK_DIR="${CYLC_WORKFLOW_RUN_DIR}/work"
-    CYLC_TASK_CYCLE_POINT="$(cut -d '/' -f 1 <<<"${CYLC_TASK_JOB}")"
-    CYLC_TASK_NAME="$(cut -d '/' -f 2 <<<"${CYLC_TASK_JOB}")"
-    export CYLC_TASK_NAME CYLC_TASK_CYCLE_POINT ISODATETIMEREF
+    export CYLC_TASK_CYCLE_POINT="${CYLC_TASK_JOB%%/*}"
+    export CYLC_TASK_NAME="${CYLC_TASK_JOB#*/}"
+    CYLC_TASK_NAME="${CYLC_TASK_NAME%/*}"
     if [[ "${CYLC_CYCLING_MODE}" != "integer" ]]; then  # i.e. date-time cycling
-        ISODATETIMECALENDAR="${CYLC_CYCLING_MODE}"
-        ISODATETIMEREF="${CYLC_TASK_CYCLE_POINT}"
-        export ISODATETIMECALENDAR ISODATETIMEREF
+        export ISODATETIMECALENDAR="${CYLC_CYCLING_MODE}"
+        export ISODATETIMEREF="${CYLC_TASK_CYCLE_POINT}"
     fi
     # The "10#" part ensures that the submit number is interpreted in base 10.
     # Otherwise, a zero padded number will be interpreted as an octal.
-    export CYLC_TASK_SUBMIT_NUMBER=$((10#$(cut -d '/' -f 3 <<<"${CYLC_TASK_JOB}")))
+    export CYLC_TASK_SUBMIT_NUMBER="$((10#${CYLC_TASK_JOB##*/}))"
```

Comment:
Huh, I wonder why we weren't just using bash variable substitution in the first place?  (`git blame` pings @oliver-sanders for the first two cases above, in mid 2019, but those were probably adapted from the last case, Matt two years earlier).